app/services/unified_price_service.py

```
@@ -262,21 +262,180 @@ async def _fetch_batch_from_source(self, symbols: List[str], source: PriceSource
+                    keys = await self.redis.keys(pattern)
+                    for key in keys:
+                        try:
+                            cached_data = await self.redis.hgetall(key)
+                            if not cached_data:
+                                # Try as simple string
+                                cached_val = await self.redis.get(key)
+                                if cached_val:
+                                    try:
+                                        import json
+                                        cached_data = json.loads(cached_val)
+                                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)